### ISSUE TITLE
How to disable drag feature?

### ISSUE THREAD

**Issue by Anandks1993** [markusenglund/react-switch#102]
Hi,

I need to disable the drag feature on the switch. Is there any way to achieve this?

Thank you in advance.

**Comment by markusenglund** [comment]
No, and it probably won't be supported anytime soon. I recommend a different library like react-toggle.
